portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,20 @@ function addRandomFunFacts() {
   const funFactContainer = document.getElementById('funFacts-container');
   funFactContainer.innerText = funFact;
 }
+
+async function showRandomGreets() {
+    console.log("testing testing");
+  const responseFromServer = await fetch('/random-greet');
+  const textFromResponse = await responseFromServer.json();
+
+  const greetContainer = document.getElementById('greet-container');
+  greetContainer.appendChild(
```

Comment:
If you call this method more than once, you'll end up with multiple greetings because you're appending the element to the container. Is that ok or do you want to set the `innerHTML` of the greetContainer?

Reply:
Good idea, definitely didn't consider that!